client/src/locales/en-US.json

```
@@ -888,8 +888,8 @@
             "joinedOn": "Joined On"
         },
         "currentUser": "(You)",
-        "inviteUser": "Invite a user",
-        "itemCount": "No user | One user | {count} users",
+        "inviteUser": "Invite a member",
+        "itemCount": "No member | One member | {count} members",
```

```suggestion
        "itemCount": "No members | One member | {count} members",
```

0 / any is plural in english